### DISCUSSION TITLE
MVP#3-UnqualifiedQuery-GeneSequenceVariantBiologicalProcess-to-Disease

### DISCUSSION THREAD

**Discussion by karafecho** [NCATSTranslator/Clinical-Data-Committee-Tracking-Voting#20] [Ideas]
This discussion thread is intended to promote discussion among the TCDC regarding the MVP#2 templated query and approaches for incorporating clinical knowledge into inferred (creative) mode responses:

_What {gene, sequence variant, biological process} [contributes to] [disease]?_

**Comment by karafecho** [comment]
This query is one that I proposed and envisioned as an AOP or Adverse Outcome Pathway, with DILI or drug-induced liver injury serving as an example 'disease'. However, the query itself has applicability to any disease or drug/chemical-related adverse outcome.

Here are a few additional thoughts that I included with my proposal:

_Given a disease, find genes, sequence variants, and/or biological processes that contribute to the disease or manifestation of disease or severity/chronicity of disease. Considerations: new predicate, could be qualified (e.g., severity, chronicity), use of underrepresented/underused Biolink category/entity type (biolink:SequenceVariant, biolink:BiologicalProcess), ‘ground truth’ or suspected genes are established for DILI (HLA-1, HLA-II, ERAP2, EXOCYST4 [EXOCYST3], PTPN22)._

**Comment by karafecho** [comment]
@marcdubybroad @bill-baumgartner : From what I gathered during today's TAQA call (and I admittedly didn't hear everything, as I stepped out for a few minutes), it sounds like both of your KPs will be able to contribute knowledge on both genes and variants to MVP#3. Is that correct?

**Reply by bill-baumgartner** [reply]
Hi @karafecho, yes, we have unqualified `gene-contributes_to-disease` data mined from the literature. We also have qualified assertions where the gene is noted as either a gain-of-function variant, or loss-of-function variant. The qualified data is in the process of being deployed and should be available in the coming days.

**Reply by marcdubybroad** [reply]
Hi @karafecho , we can contribute gene/disease information. We are looking to qualify some of our results with loss of function variant qualifiers. We currently don't have any variant data loaded, just gene data that is generated for meta analyzing variant p-values.

**Comment by karafecho** [comment]
Great! Thanks for the updates, Bill and Marc. I knew about the unqualified gene-disease data, but I was not aware of the qualified variant data.
